scaleway/helpers_k8s.go

```diff
@@ -57,11 +58,43 @@ func k8sAPIWithRegionAndID(m interface{}, id string) (*k8s.API, scw.Region, stri
 	return k8sAPI, region, ID, err
 }
 
+func waitK8SClusterPoolRequired(k8sAPI *k8s.API, region scw.Region, clusterID string) error {
```

Comment:
Can use a single function to reach a desired list of state 
 `waitK8SClusterPool(k8sAPI *k8s.API, region scw.Region, clusterID string, desiredState ... k8s.PoolStatus)`